spirekey/src/app/(embedded)/embedded/notification/page.tsx

```diff
@@ -60,38 +61,33 @@ export default function SidebarSign() {
     setIsMinimized(false);
   };
 
-  const showSidebarNotifications = () => {
-    publishEvent('show-notifications-sidebar');
-  };
-
-  useTheme({ lockedTheme: 'light' });
-
   return (
-    <div style={{ width: '100%', height: '100%' }}>
-      <div
-        className={styles.wrapper}
-        onClick={isMinimized ? maximize : () => {}}
-        style={
-          {
-            '--card-progress': `50%`,
-            '--card-progress-color-start': colorStart,
-            '--card-progress-color-end': colorEnd,
-          } as React.CSSProperties
-        }
+    <div
+      className={styles.wrapper}
+      style={
+        {
+          '--card-progress': `50%`,
+          '--card-progress-color-start': colorStart,
+          '--card-progress-color-end': colorEnd,
+        } as React.CSSProperties
+      }
+    >
+      <button
+        className={styles.unstyledButton}
```

Comment:
since this is the close button using plain html button without text in there, should this have aria labels?

Reply:
yes, it should! I'm going to try replacing it with a kode-ui button as well